README.md

```diff
@@ -212,18 +205,18 @@ The plugin fires events at various points during its lifecycle which users can h
 
 It will always supply the current buffer in the `data.saved_buffer`
 
-An example to always print the file name before a file is getting saved (use `:messages` if the execution message swallows the print):
+An example to print a message with the file name after a file got saved:
 
 ```lua
 local group = vim.api.nvim_create_augroup('autosave', {})
 
 vim.api.nvim_create_autocmd('User', {
-    pattern = 'AutoSaveWritePre',
+    pattern = 'AutoSaveWritePost',
     group = group,
     callback = function(opts)
         if opts.data.saved_buffer ~= nil then
             local filename = vim.api.nvim_buf_get_name(opts.data.saved_buffer)
-            print('We are about to save ' .. filename .. ' get ready captain!')
+            print('File ' .. filename .. ' got saved successfully!')
```

Comment:
Let's use the same string here in the example the old execution message used to be: `"AutoSave: saved at " .. vim.fn.strftime("%H:%M:%S")`. Or reformat it to `auto-save: ...`. WDYT?

Reply:
I changed it to `'AutoSave: saved ' .. filename .. ' at ' .. vim.fn.strftime('%H:%M:%S'))`, because I want to show that we get the buffer handle in the `opts` and how we can get the name of the file with it.